documentation/man/features/profiler.rst

```diff
@@ -0,0 +1,43 @@
+==============
+kw-profiler
```

Comment:
squash this commit in the one that introduces the profile.

Reply:
Done